src/modules/picklist/PickListComponent.js

```diff
@@ -220,7 +134,7 @@ export class PickListContainer extends React.Component {
   })
 
   moveModal = (items) => (to, toName, onSuccess) => {
-    const isNode = this.isTypeNode();
+    const isNode = this.props.isTypeNode();
```

Comment:
Forgot to send this props AS argument to isTypeNode. Or maybe find another way to do this ? E.g. Share a helper function between the comp and hoc?